tools/install/CreateInstallers.bat

```diff
@@ -25,6 +25,7 @@ IF EXIST %cwd%\..\..\bin\%OPT_Platform%\%OPT_CONFIGURATION%\Revit_2016 (
 
 robocopy %cwd%\..\..\bin\%OPT_Platform%\%OPT_CONFIGURATION%\nodes %cwd%\temp\bin\nodes *.dll *.xml /e
 robocopy %cwd%\..\..\bin\%OPT_Platform%\%OPT_CONFIGURATION%\extensions %cwd%\temp\bin\extensions *.xml /e
+robocopy %cwd%\..\..\bin\%OPT_Platform%\%OPT_CONFIGURATION%\viewExtensions %cwd%\temp\bin\viewExtensions *.dll *.xml /e
```

Comment:
Here I'm not sure. I copy `.dll` and `.xml`. But in `extensions` we don't copy `.dll` files.